x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -639,21 +639,28 @@ through either the
 NOTE: You cannot use <<mapping-roles-file,role mapping files>>
 to grant roles to users authenticating via SAML.
 
-This is an example of a simple role mapping that grants the `kibana_user` role
+This is an example of a simple role mapping that grants the `siem_view` role
 to any user who authenticates against the `saml1` realm:
 
 [source,console]
 --------------------------------------------------
 PUT /_security/role_mapping/saml-kibana
 {
-  "roles": [ "kibana_user" ],
+  "roles": [ "siem_view" ], <1>
```

Comment:
I think this will be confusing for some users. Since we're talking about a role that is not there, I'd opt for a generic role name role and a generic explanation below that they should create the role - as opposed to tying it (even as an example) to a product or to Kibana feature privileges

cc @legrego